app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -222,6 +222,7 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         if (!shouldStartOnHome() &&
             shouldNavigateBrowserFragmentOnCouldStart(savedInstanceState)
         ) {
+            NavGraphProvider.blockForNavGraphInflation(navHost.navController)
```

Comment:
Why are these new lines necessary? In the initial PR, we added lines like these but they were unnecessary and only ended up slowing down the browser unnecessarily in some cases.

Reply:
Good catch, I'll remove